src/test/java/javaslang/jackson/datatype/seq/ListTest.java

```diff
@@ -66,4 +74,55 @@ public void testJsonTypeInfo() throws IOException {
         Assert.assertEquals("hello", restored.f.head().type);
     }
 
+    @XmlRootElement(name = "xmlSerialize")
+    private static class JaxbXmlSerializeJavaslang {
+        @XmlElementWrapper(name = "transitTypes")
+        @XmlElement(name = "transitType")
+        public List<Integer> transitTypes = List.of(1, 2, 3);
+    }
+
+    @XmlRootElement(name = "xmlSerialize")
+    private static class JaxbXmlSerializeJavaUtil {
+        @XmlElementWrapper(name = "transitTypes")
+        @XmlElement(name = "transitType")
+        public java.util.List<Integer> transitTypes = new java.util.ArrayList<>();
+
+        public JaxbXmlSerializeJavaUtil() {
+            transitTypes.add(1);
+            transitTypes.add(2);
+            transitTypes.add(3);
+        }
+    }
+
+    @Test
+    public void testJaxbXmlSerialization() throws IOException {
+        String javaUtilValue = xmlMapperJaxb().writeValueAsString(new JaxbXmlSerializeJavaslang());
+        Assert.assertEquals(xmlMapperJaxb().writeValueAsString(new JaxbXmlSerializeJavaUtil()), javaUtilValue);
+    }
+
+    @JacksonXmlRootElement(localName = "xmlSerialize")
+    private static class XmlSerializeJavaslang {
+        @JacksonXmlElementWrapper(localName = "transitTypes")
+        @JsonProperty("transitType")
+        public List<Integer> transitTypes = List.of(1, 2, 3);
+    }
+
+    @JacksonXmlRootElement(localName = "xmlSerialize")
+    private static class XmlSerializeJavaUtil {
+        @JacksonXmlElementWrapper(localName = "transitTypes")
+        @JsonProperty("transitType")
+        public java.util.List<Integer> transitTypes = new java.util.ArrayList<>();
+
+        public XmlSerializeJavaUtil() {
+            transitTypes.add(1);
+            transitTypes.add(2);
+            transitTypes.add(3);
+        }
+    }
+
+    @Test
+    public void testXmlSerialization() throws IOException {
+        String javaUtilValue = xmlMapper().writeValueAsString(new XmlSerializeJavaslang());
+        Assert.assertEquals(xmlMapper().writeValueAsString(new XmlSerializeJavaUtil()), javaUtilValue);
```

Comment:
Something wrong with deserialization of XML-source. Here I added these lines:
```java
XmlSerializeJavaslang restored = xmlMapper().readValue(javaUtilValue, XmlSerializeJavaslang.class);
Assert.assertEquals(restored.transitTypes.size(), 3);
```
Process fails with error:
```
com.fasterxml.jackson.databind.JsonMappingException: Can not deserialize instance of java.lang.Integer out of FIELD_NAME token
 at [Source: java.io.StringReader@53aad5d5; line: 1, column: 51] (through reference chain: javaslang.jackson.datatype.seq.XmlSerializeJavaslang["transitTypes"])

	at com.fasterxml.jackson.databind.JsonMappingException.from(JsonMappingException.java:148)
	at com.fasterxml.jackson.databind.DeserializationContext.mappingException(DeserializationContext.java:857)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer._parseInteger(StdDeserializer.java:460)
	at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$IntegerDeserializer.deserialize(NumberDeserializers.java:309)
	at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$IntegerDeserializer.deserialize(NumberDeserializers.java:287)
	at javaslang.jackson.datatype.deserialize.ArrayDeserializer.deserialize(ArrayDeserializer.java:52)
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:520)
	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:101)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:258)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:125)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3736)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2726)
	at javaslang.jackson.datatype.seq.ListTest.testXmlSerialization(ListTest.java:129)
       .....
```